packages/webb-ui-components/src/containers/KeygenTable/KeygenTable.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  ColumnDef,
+  ColumnFiltersState,
+  createColumnHelper,
+  getCoreRowModel,
+  getFacetedMinMaxValues,
+  getFacetedRowModel,
+  getFacetedUniqueValues,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  PaginationState,
+  Table as RTTable,
+  useReactTable,
+} from '@tanstack/react-table';
+import {
+  Avatar,
+  AvatarGroup,
+  Button,
+  CardTable,
+  Collapsible,
+  CollapsibleButton,
+  CollapsibleContent,
+  Filter,
+  KeyValueWithButton,
+  Slider,
+  Table,
+} from '@webb-dapp/webb-ui-components/components';
+import { fuzzyFilter } from '@webb-dapp/webb-ui-components/components/Filter/utils';
+import { Authority } from '@webb-dapp/webb-ui-components/components/KeyStatusCard/types';
+import { fetchKeygenData } from '@webb-dapp/webb-ui-components/hooks';
+import { KeygenType } from '@webb-dapp/webb-ui-components/types';
+import { useEffect, useMemo, useState } from 'react';
+
+const columnHelper = createColumnHelper<KeygenType>();
+
+const columns: ColumnDef<KeygenType, any>[] = [
```

Comment:
I think it's reasonable to add tooltips to the column headers so users can quickly get an understanding of the data.

Reply:
Header tooltip content:

**Height:** Number of blocks
**Session:** A session is a period that has a constant set of validators. Validators can only join or exit the validator set at a session change.
**Key:** The public key of the DKG protocol that is currently active.
**Signature Threshold:** The ‘t’ in (t-out-of-n) threshold signatures used in the DKG signing system. Required of DKG authorities to generate signatures.
**Authority Set:** A set of DKG authorities